doc/development_guide/contributor_guide/governance.rst

```diff
@@ -0,0 +1,48 @@
+==============
+ Governance
+==============
+
+How to become a triager ?
+-------------------------
+
+Contribute for more than 3 releases consistently. Do not be too opinionated, follow the code of conduct without
+requiring emotional works from the maintainers. It does not mean that disagreements are impossible, only that arguments
+should stay technical and reasonable so the conversation is civil and productive. Get unanimous approval or neutral
+agreement from current maintainers.
+
+How to become a maintainer ?
+----------------------------
+
+Contribute two big code merge requests over multiple releases (for example like one checker in 2.13 and one
+complicated bug fixes in 2.14). Otherwise contributing for more than 3 releases consistently with
+great technical and interpersonal skills. Become an expert in a part of the code that is not maintained
+well at the moment (if we feel we can't merge something without your review, you're going to be able to
+merge those yourself soon). Get unanimous approval or neutral agreement from current maintainers.
+
+
+How to get admin rights ?
+-------------------------
+
+Contribute for several hundreds of commits over a long period of time with excellent interpersonal skills
+and code quality. Get unanimous approval or neutral agreement from current admins.
+
+
+How are decisions made ?
+------------------------
+
+Everyone is expected to follow the code of conduct. pylint is a do-ocracy / democracy.
+You're not allowed to behave poorly because you contributed a lot. But if
+you're not going to do the future maintenance work, your valid opinions might not be
+taken into account by those that will be affected by it.
+
+When something is not consensual users, maintainers, and admins discuss until an agreement is reached.
+
+Depending on the difficulty of the discussion and the importance of a fast resolution, a decision can be taken:
+
+- Unanimously between discussion participants, contributors and maintainers (preferably)
+
+- By asking discussion participants for their opinions with an emoji survey in the issue and then using the majority
+  if no maintainers feel strongly about the issue.
+
+- By asking all users for their opinions in a new issue that will be pinned for multiple months before taking the decision
+  if two maintainers feel strongly on an opposite side of the issue.
```

Comment:
I think this could also be something an admin needs / could decide.
Similar to the Python steering council, let the maintainers find a solution. If that's not possible, step in and make a decision.